src/scss/includes/panels/categories.scss

```diff
@@ -6,47 +6,48 @@
 
 .category__panel__pj {
   width: 100%;
-  height: 50px;
-  padding: 0 20px;
-  display: flex;
-  line-height: 18px;
-  align-items: center;
-  justify-content: space-between;
-
-  &_title {
-    font-weight: normal;
-    color: $primary_text;
-    background: url('../images/pagesjaunes.svg') no-repeat left center;
-    background-size: 18px;
-    padding-left: 24px;
-  }
-
-  &_partnership {
-    display: inline-flex;
-    align-items: center;
-    font-size: 12px;
-    color: $secondary_text;
-
-    .icon-icon_info {
-      font-size: 16px;
-      margin-right: 4px;
-    }
-  }
+  height: 36px;
+  position: absolute;
+  bottom: 0;
+  border-top: 1px solid $grey-light;
+  background: $background;
+  line-height: 36px;
+  text-align: center;
+  font-size: 14px;
+  color: $grey-semi-darkness;
+  border-radius: 0 0 12px 12px;
 }
 
 .category__panel__items {
   animation: appear 600ms forwards;
 }
 
 .category__panel__error {
-  padding: 16px 16px 32px 16px
+  padding: 16px 16px 32px 16px;
+
+  & ~ .category__panel__pj {
```

Comment:
In the React component you chose not to render the block in case of error, so this rules seems unnecessary.